### ISSUE TITLE
Cannot download mods [0.11.3] [Linux]

### ISSUE THREAD

**Issue by Develon5543** [tModLoader/tModLoader#633]
### Description
When attempting to download mods on the mod browser, the progress bar hangs at 0% and the download never completes.

### Steps to Reproduce
1. Start Terraria and open the mod browser.
2. Attempt to download any mod.

### Expected behavior
The mod should be downloaded normally.

### Actual behavior
The mod is never downloaded; the progress hangs at 0%.

### Reproduction frequency
Always.

### Versions
tModLoader v0.11.3

### Mods
No mods are enabled or loaded.

### Additional Information
I'm using Linux and the logs just say this:
`[14:04:53] [Threadpool worker/DEBUG] [tML]: Web Request: https://cdn.discordapp.com/attachments/284077567986761730/604707142926204928/TheGodhunter.tmod`
`[14:25:02] [1/WARN] [tML]: UIDownloadProgress was deactivated but download [/home/develon/.local/share/Terraria/ModLoader/Mods/TheGodhunter.tmod] was still present.`
Also, I can download that link from my browser, but not Terraria. It happens with every mod I've tried. Also also, the file it mentions above doesn't exist while downloading or after cancelling the download. So I'm not sure what that's about.

**Comment by JavidPack** [comment]
Full log would help, even if you don't think it says anything interesting.

**Comment by Develon5543** [comment]
[Here](https://pastebin.com/fHmW2SAr) you go. This is from me starting the game, trying to download a mod, and then cancelling after some time.

**Comment by Develon5543** [comment]
Hey guys. I have [another log](https://pastebin.com/rF4NKT5k) for you. This one is after trying to download a mod, then cancelling, then scrolling through the mod browser.

**Comment by Jofairden** [comment]
Hmm, this seems similar to the china people having issues.
Can you access these links normally? (Though the browser)
For example: https://i.imgur.com/BlNk55N.png
And: https://cdn.discordapp.com/attachments/284077567986761730/604877321819914252/OmegasArsenal.tmod

**Comment by Develon5543** [comment]
Yeah, I can access those in my browser. I tried to download a mod again just now to see if maybe it would work, but I still get the same result. I think the last time it worked was 0.11 beta 6 and 0.11. I haven't tried 0.11.1 or 0.11.2 though.

**Comment by Develon5543** [comment]
Alright I tried fully updating my computer and everything. This time around, I let it run for 44 minutes on the download screen, but nothing happens and there's no indication in the logs that something is wrong. I just don't know what to do at this point..

**Comment by nedart** [comment]
I'm having the same issue on Linux

**Comment by JavidPack** [comment]
Try 0.11.4 nedart

**Comment by dfso** [comment]
> Try 0.11.4 nedart

same bug occurs in version 0.11.4

**Comment by Daswf852** [comment]
I think dfso's comment should be added in the linux README file

**Comment by danielpza** [comment]
can confirm it works on fedora: `sudo dnf install mono-core mono-tools mono-addins`

**Comment by Lesutachi** [comment]
```
E: Unable to locate package mono-core
E: Unable to locate package mono-tools
E: Unable to locate package mono-addins
```
What can I do on Mint / Ubuntu?
There's mono-tools-gui and mono-tools-devel, and mono-utils. Tried installing mono-tools-devel and mono-utils but it didn't work yet. I'm holding off installing mono-complete, but if that does it, I might as well try. It's tiresome to download the mods 1 by 1 from logs at the console.

**Comment by thundergoth** [comment]
This bug is still present for me.

**Comment by Mirsario** [comment]
Use system Mono, or manual downloads from http://javid.ddns.net/tModLoader/DirectModDownloadListing.php
This bug will be fixed for certainly in 1.4.
